src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossSummary.scala

```diff
@@ -116,8 +118,40 @@ object TreeLossSummary {
             summary.totalArea += areaHa
             summary.totalGainArea += gainArea
 
-            // exclusive threshold because of the tcd class used (unlike other packages)
-            if ((tcd2000 > thresholds.head && tcdYear == 2000) || (tcd2010 > thresholds.head && tcdYear == 2010)) {
+            // Specifically for TCD=0. Otherwise, values for TCD=0 are under-estimated.
+            if ((thresholds.head == 0 && tcdYear == 2000) || (thresholds.head == 0 && tcdYear == 2010)) {
+
+              if (loss != null) {
```

Comment:
I'm a little confused what this is for, this code block seems to be exactly the same to one for non-zero thresholds? Do you mean to just make one if state, something like:
`if (thresholds.head == 0 or tcd2000 > thresholds.head)`?

Reply:
We had a follow-up Slack discussion about how this suggestion doesn't produce the correct results either. I think these threshold checks are where the non-flux model results are diverging from annualupdate_minimal/dashboards.